### PR TITLE
Add LC links to Submit a Small Contribution section

### PR DESCRIPTION
**Pull request by libcce**

### CODE REVIEW

File: _extras/checkout.md [hunk 1]
```diff
@@ -58,12 +58,11 @@ Trainees must make a contribution to a lesson's content, exercises, or instructo
...
 
 Each lesson has a CONTRIBUTING.md file in its repository on GitHub. This file explains best practices for contributing
-to that lesson. You can access a lesson's GitHub repository by clicking on the GitHub kitty icon in the "Repository" column for that
-lesson on the lesson page. For Data Carpentry this is [www.datacarpentry.org/lessons/]({{ site.dc_site }}/lessons/). For Software Carpentry this is
-[www.software-carpentry.org/lessons/]({{ site.swc_site }}/lessons/). Please follow the guidelines in the appropriate CONTRIBUTING.md file when making lesson
+to that lesson. You can access a lesson's GitHub repository by clicking on the GitHub kitty icon in the "Repository" column for that lesson on the lesson page. For Data Carpentry this is [www.datacarpentry.org/lessons/]({{ site.dc_site }}/lessons/). For Software Carpentry this is
+[www.software-carpentry.org/lessons/]({{ site.swc_site }}/lessons/). For Library Carpentry this is [www.software-carpentry.org/lessons/]({{ site.lc_site }}/lessons/). Please follow the guidelines in the appropriate CONTRIBUTING.md file when making lesson
```
Comment (tracykteal) [inline _extras/checkout.md]: Is this URL correct for Library Carpentry? It has software-carpentry.org in it.

**Comment by tracykteal** [comment]
Thanks @libcce. Added a comment checking if the URL is correct.

**Comment by tracykteal** [comment]
Also, for Library Carpentry do you want to require all the lesson contributions come through github or also through email? For Data Carpentry we also allow them through email, because instructors aren't required to know github to teach DC.

File: _extras/checkout.md (hunk 1 above)
Comment (libcce) [reply _extras/checkout.md]: So much for doing this quickly. Yes, I need to change this. Thanks for catching it!

**Comment by libcce** [comment]
I opted for GitHub. Almost all of our contributions come via GitHub and my contact info is on the LC site if people want to contact me directly. Seems the best way? But I defer to you :)

**Comment by tracykteal** [comment]
Given that knowing github also isn't a requirement for LC instructors, it would be a good idea to include email as an option for LC checkout. Since we already have that workflow available, it would work well and @sheraaron says it's working fine. So, then instead of changing the Software Carpentry sentence, you can change the Data Carpentry one to say "Contributions to Data Carpentry and Library Carpentry materials may be submitted through GitHub or by [email](mailto:{{ site.email }})."

**Comment by kariljordan** [comment]
+1 to the e-mail option

**Comment by libcce** [comment]
Via the site email, ok, that makes sense. Thanks! I made the update but also added bullets/pointers to the GitHub repos which I thought would be helpful since we mention that contributions can be made via GitHub.

**Comment by tracykteal** [comment]
There's so many other things in our github organizations than just lessons, that directing people to that top level URL is unfortunately more confusing than helpful. The links in the following paragraph direct people to the lessons pages, which then you can use to get the github pages. Having those links clearer is a good idea though. Instead of having them in those sentences, we could pull them out as bullet points there.

**Comment by libcce** [comment]
Thanks! Pulled them out as bullet points now.

**Comment by tracykteal** [comment]
Thanks! Looks good.

**Pull request merged**